.editorconfig

```diff
@@ -0,0 +1,18 @@
+[*]
+charset=utf-8
+end_of_line=crlf
+insert_final_newline=true
+indent_style=space
+indent_size=2
+
+ [{.eslintrc,.nycrc,.babelrc,.stylelintrc,jest.config,.luisrc,*.json,*.jsb3,*.jsb2,*.bowerrc}]
+indent_style=space
+indent_size=2
+
+ [*.js.map]
+indent_style=space
+indent_size=2
+
+ [{.analysis_options,*.yml,*.yaml}]
+indent_style=space
+indent_size=2
```

Comment:
Aren't these duplicative with the global rule above?

Reply:
Yes they are duplicates and can be removed.

Reply:
Removed all except the global one.